compiler_opt/tools/extract_ir.py

```diff
@@ -30,6 +30,7 @@
 import json
```

Comment:
also update in the comment on top about the information of your thinlto lld case